test/test_helper.rb

```diff
@@ -13,7 +13,12 @@
 
 ActiveRecord::Migration.verbose = false
 ActiveRecord::Base.logger = Logger.new(nil)
-ActiveRecord::Migrator.migrate(File.expand_path('../dummy/db/migrate', __FILE__))
+if Rails.version == '5.2.0'
+  mc = ActiveRecord::MigrationContext.new(File.expand_path('../dummy/db/migrate', __FILE__))
```

Comment:
not really using `mc`.  Just write it as
```
ActiveRecord::MigrationContext.new(File.expand_path('../dummy/db/migrate', __FILE__)).migrate
```

Reply:
fixed in latest commit